### DISCUSSION TITLE
Make members invisible to undeads when wearing carved pumpkin

### DISCUSSION THREAD

**Discussion by srnyx** [commandgeek/geeksmp#92] [Ideas]
Sorta like the thing with enderman when a player is wearing a carved pumpkin.
Any alive players will be invisible to undeads (still visible to other alive players).
This could create a lot of interesting PVP techniques and other stuff.

**Comment by techred32** [comment]
Maybe there could be a craft able item involving nether stars and other rare items that if held in the offhand and crouching could make an alive player invisible. I think carved pumpkin is too op. It’s easy to acquire and there are many texture packs that make the effects disappear. I think a custom item, while harder to make, would be cooler. Just my opinion.

**Reply by techred32** [reply]
Also, since it must be in the offhand a totem of undying couldn’t be used at the same time.

**Reply by techred32** [reply]
I feel like this would be more balanced

**Reply by srnyx** [reply]
I wasn't really thinking of the overlay being a downside cause, like you said, a resourcepack can easily disable it.
I was mainly focusing on the fact that you wouldn't be able to wear a helmet.

But, I definitely like the idea of a custom item and taking the offhand slot (although it is more of an impact if it took the helmet slot cause totems are only used when you "die," and have a limited supply). I also like how it would require the player to sneak, cause then it's like they are actually "sneaking around."
So, in the end, I think it'd be better to have it be a custom item held in the offhand that requires you to sneak for it to activate. Maybe it could even have durability (but can be enchanted with Unbreaking and Mending), where it would go down once every second while using it.
The recipe could be:
| 1 | 2 | 3 |
| :---: | :---: | :---: |
| C | N | C |
| Z | I | S |
| C | N | C |

`C` = Conduit
`N` = Nether Star
`Z` = Zombie Skull
`I` = Potion of Invisibility
`S` = Skeleton Skull
And it could be named something like `Totem of Dying` (but we wouldn't be able to use the totem item)

This is definitely a super unique concept and I'd love to see this implemented (I may be able to help with some of it, but won't be able to do it all by myself as it's too complex for me).

**Comment by Lianecx** [comment]
Only make them invisible to undeads in a certain range (if they get close they should see you again) and depending on the head they wear (skeleton or zombie) they are only visible to that type of zombie. Thats how it works in mc if you wear zombie or skeleton heads.

**Reply by srnyx** [reply]
Not sure about the proximity range cause then it wouldn't be that useful. And maybe it could still be possible to see the particles when the player walks, so it's like invisibility (except you can't see armor). Or we could just make it be invisibility except it hides the item itself.
I do like the thing that depends on which head you wear, that way the other type can still see you. *I didn't know that was vanilla.* It could use the same recipe I put above except switch out the zombie skull for a skeleton or vise-versa.
